tests/test_enterprise/api/test_views.py

```diff
@@ -1171,6 +1172,7 @@ def test_enterprise_catalog_contains_content_items_no_query_params(self):
         assert 'course_run_ids' in message
         assert response.status_code == 400
 
+    @pytest.mark.skip(reason='Catalog course run detail endpoint now lives in enterprise-catalog service')
```

Comment:
These detail endpoints don't live in enterprise catalog yet though

Reply:
Damn.. okay that's my bad. Working on test fixes now